files/entry.js

```diff
@@ -1,6 +1,7 @@
 import { installPolyfills } from '@sveltejs/kit/node/polyfills';
 import { Server } from 'SERVER';
 import { manifest } from 'MANIFEST';
+import { split_cookies_from_headers } from './headers';
```

Comment:
Nit: we use camel case everywhere else in the adapter
```suggestion
import { splitCookiesFromHeaders } from './headers';
```